### ISSUE TITLE
Review Dialect: Tagalog (Python)

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#86]
## Review Tagalog Language Translations for the Python Programming Language

Version 1 of Legesher only includes translations for the Python programming language. In order to add this language to the project, complete the steps detailed in [the translation guide](https://legesher.readme.io/docs/translation-guide). 

**Reviewing** a translation is extremely important as we want to best understand the culture and context behind the words we select as keywords for the programming languages. The more perspectives the better! 👀 

>⚠️ Note: this issue is to review the Python keywords and builtin functions only. Any additional translation reviews are encouraged, but not necessary. 

**To complete this issue**:
- [ ] Navigate to the _Python_ section of the dialect's `locale/DIALECT.yml` file
- [ ] Go through each of the translations for the Python keywords and built in functions adding commentary as comments
  - Leave a `✅`after a translation if you agree that's the right word to give context to the keyword
  - Leave a `❓` after a translation if you think there may be a better translation / form of the translation
  - Leave a  `🤔` after a word that you're not sure if there is a translation
- [ ] Create a pull request with changes into branch titled `tagalog-translation` on this repo
- [ ] In your pull request, reference the original pull request that added your dialect. 

### Reviewers Needed
In order to add the translations into our database to use, the translations will need to be approved and accepted by 3 additional native / fluent speakers. If you would like to be one of the reviewers for this language, please mention in this issue! :smile: Also feel free to loop in a friend that might be able to help with the translations!

The translated word needs to have `# ✅✅✅` in the comments after it in order to be accepted. Please take your time in being thorough with these translations. Think back when you were learning how to program and how context of words help you bridge together meaning and functionality. 

**Review Status:**
- [x] ~`Add New Dialect` Issue~
- [x] Initiator: @Kcmvillarino (#85)
- [ ] Reviewer 1: 
- [ ] Reviewer 2: 
- [ ] Reviewer 3:

**Comment by jayehernandez** [comment]
Hi @madiedgar, I'd love to take a shot at reviewing this, will open up a PR!

**Comment by madiedgar** [comment]
That would be amazing @jayehernandez, and yes please add in the Python built-in functions in your PR as well! 😄

**Comment by madiedgar** [comment]
Hey @kcmv 👋 , thanks so much for helping with adding the initial Tagalog translation. Do you know anyone else who might be interested in contributing to this? Thank you 👍 

Hey @jayehernandez 👋 , are you still interested in helping with this issue. Would be a great help! 👍

**Comment by kcmv** [comment]
Wow, small world @jayehernandez. Well anyway, i will look into this and ask
some friends.

**Comment by jayehernandez** [comment]
Thanks for the mention on this @madiedgar! I think I was able to help out with #95. 😁 @kcmv hello!!

**Comment by madiedgar** [comment]
I apologize @jayehernandez, thanks so much for your contribution in #95 👍

**Comment by joshuiaclan** [comment]
Hi @madiedgar, Can I take a shot in this?

**Comment by madiedgar** [comment]
Yes please @joshuiaclan 👍 Let me know if you have any questions!
